### ISSUE TITLE
Some streams don't play in v12 nor v13 but play fine in v11.1 and lower.

### ISSUE THREAD

**Issue by what-a** [4gray/iptvnator#253]
Here are some examples of streams that work in version 11.1 and lower, but do not work in version 12 nor 13:
https://cdn1-9p.teleuptv.net/f4441462-c21e-4269-9721-84d8cbd93ebf/index.m3u8
https://cdn1-9p.teleuptv.net/fad91b41-322d-4d6e-a427-551e8d8199bb/index.m3u8

My guess is that it has something to do with the User Agent setting not working properly in v12 nor v13.

**Comment by 4gray** [comment]
Thanks for the user agent hint. 
That's most likely the problem, as the user agent defined at playlist level is now not taken into account. Unfortunately I missed this detail during refactoring, I'll try to fix it soon.

**Comment by what-a** [comment]
This bug still exists in version 0.16.0
Also, in 0.16.0 the playlist user agent details and the "auto-update" checkmark vanish upon refreshing a playlist.

**Comment by luis-antunes** [comment]
Yeah, I ran into the same issue on the MacOS (IPTVnator-0.16.0.dmg) version using Xtreme Code but was able to successfully see the custom user-agent being correctly used when the playlist was added using "Adding from URL".
